client/components/forms/form-country-select/index.jsx

```diff
@@ -20,6 +20,9 @@ export class FormCountrySelect extends Component {
 		disabled: PropTypes.bool,
 		onChange: PropTypes.func,
 		translate: PropTypes.func.isRequired,
+		ariaLabelledBy: PropTypes.string,
+		ariaDescribedBy: PropTypes.string,
+		selectedValue: PropTypes.string,
```

Comment:
You should mention in the PR description that this also adds a `selectedValue` prop and changes the component to be a controlled component. That is, once you set `selectedValue`, you'll have to make sure that the `onChange` function changes the `selectedValue`.

It might also be worth considering this prop's name. Maybe just `value` would be more accurate?